### ISSUE TITLE
Improve typescript support for methods { event, purchase, ... }

### ISSUE THREAD

**Issue by Maucin** [MatteoGabriele/vue-gtag#210]
### Description

File `vue-gtag.d.ts` does not contain export for methods like: `event`, `purchase` ... . When I try to import in the `vuex` those methods I get message `TS2614: Module '"vue-gtag"' has no exported member 'purchase'. Did you mean to use 'import purchase from "vue-gtag"' instead?`. 

Related items:
https://github.com/MatteoGabriele/vue-gtag/commit/7e1a40162d5226e91b1c37e61f629e551481b7b0 - commit with functionality
https://github.com/MatteoGabriele/vue-gtag/issues/56 - feature request

### Expected behavior

File `vue-gtag.d.ts` should contain exports for methods like: `event`, `purchase`, and so on.

### Actual behavior

There are no typings for new functionality which has been added in https://github.com/MatteoGabriele/vue-gtag/commit/7e1a40162d5226e91b1c37e61f629e551481b7b0

### Environment

Open IDE and try to use that in the project with typescript support:
`import { event, purchase } from 'vue-gtag';`

1. Operating system: doesn't matter
2. Browser and version: doesn't matter

**Comment by MatteoGabriele** [comment]
true, I must have forgotten. 
if you are familiar with type definitions, please go ahead and make a PR for this, otherwise I will pick this up as soon as I have time. 

thanks again

**Comment by Maucin** [comment]
Please take a look on https://github.com/MatteoGabriele/vue-gtag/pull/213

Cheers,
Marcin

**Comment by vmihailenco** [comment]
Is there anything specific I need to do to enable TS types (import `vue-gtag.d.ts`?)? I've installed gtag

```ts
import VueGtag from 'vue-gtag'

Vue.use(VueGtag, ...)
```

but get the error (typescript 4.1.5)

```shell
Property '$gtag' does not exist on type 'VueConstructor<Vue>'.

Vue.$gtag.event('login', { method: 'Google' })
       ^
```

**Comment by MatteoGabriele** [comment]
fixed with #213
